Plugins/Reqnroll.MSTest.Generator.ReqnrollPlugin/build/Reqnroll.MsTest.targets

```diff
@@ -33,5 +33,20 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="SetPluginParameters" BeforeTargets="UpdateFeatureFilesInProject">
+    <PropertyGroup>
+      <!-- Detect the version of MSTest.TestFramework or MSTest package and pass it to the plugin. -->
+      <!-- MSTest.TestFramework has priority if both are defined -->
+      <_MsTestFrameworkVersion Condition="'$(_MsTestFrameworkVersion)' == ''">@(PackageReference->WithMetadataValue('Identity', 'MSTest.TestFramework')->'%(Version)')</_MsTestFrameworkVersion>
```

Comment:
That version detection does not work with CPM enabled.

Reply:
Oh. Good point. Probably we need to get the resolved packages then